protocol/query/method/message/data/dataCashTransaction.json

```diff
@@ -0,0 +1,100 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "https://raw.githubusercontent.com/dedis/popstellar/master/protocol/query/method/message/data/dataPostTransaction.json",
+  "title": "Post a digital cash transaction",
+  "type": "object",
+  "properties": {
+    "object": {
+      "const": "transaction"
+    },
+    "action": {
+      "const": "post"
+    },
+    "transaction": {
+      "Version": {
+        "type": "integer",
+        "description": "The version of the transaction inputs"
+      },
+      "TxIn": {
+        "type": "array",
+        "description": "[Array[Objects]] array of output transactions to use as inputs",
+        "items": {
+          "type": "object",
+          "description": "Object representing a transaction output to use as an input for this transaction",
+          "properties": {
+            "TxOutHash": {
+              "type": "string",
+              "contentEncoding": "base64",
+              "description": "Previous (to-be-used) transaction hash"
+            },
+            "TxOutIndex": {
+              "type": "bytes",
+              "length": "4",
+              "description": "index of the previous to-be-used transaction"
+            },
+            "Script": {
+              "type": "object",
+              "description": "The script describing the TxOut unlock mechanism",
+              "properties": {
+                "Type": {
+                  "type": "string",
+                  "description": "Type of script"
+                },
+                "PubkeyHash": {
+                  "type": "string",
+                  "contentEncoding": "base64",
+                  "description": "Hash of the recipient’s public key"
+                }
+              }
+            },
+            "Sequence": {
+              "type": "bytes",
+              "length": "4",
+              "$comment": "Set as 0xFFFFFFFF"
+            }
+          }
+        }
+      },
+      "TxOut": {
+        "type": "array",
+        "description": "[Array[Objects]] array of outputs from this transactions",
+        "items": {
+          "type": "object",
+          "description": "Object representing an output of this transaction",
+          "properties": {
+            "Value": {
+              "type": "bytes",
+              "length": "8",
+              "description": "the value of the output transaction, expressed in miniLAOs"
+            },
+            "Script": {
+              "type": "object",
+              "description": "The script describing the unlock mechanism",
+              "properties": {
+                "Type": {
+                  "type": "string",
+                  "description": "Type of script"
+                },
+                "PubkeyHash": {
+                  "type": "string",
+                  "contentEncoding": "base64",
+                  "description": "Hash of the recipient’s public key"
+                },
+                "Sig": {
```

Comment:
the signature should be in the `TxIn`s and be signed by the recipients of the inputs, not the current recipient.

Reply:
Riight I swapped the script part of the TxIn with the TxOut one